doc/Extensions/SNMP-Trap-Handler.md

```diff
@@ -13,6 +13,16 @@ Install snmptrapd via your package manager.
 
 Modify startup options to include `-M /opt/librenms/mibs -m ALL`
 
+If you are using Ubuntu, you can modify the ExecStart line in the snmptrapd.service file in /lib/systemd/system/snmptrapd.service to look like the following.
```

Comment:
Standard practice is to copy snmptrapd.service to `/etc/systemd/system` and edit.  This probably isn't Ubuntu specific.

Reply:
You're correct, I changed the wording.